src/app/core/models/common/export-settings.model.ts

```
@@ -21,140 +22,155 @@ export type ExportSettingOptionSearch = {
+    // Determine the excluded date based on export Type
+    const excludedPostedAtOption = !isCoreCCCModule ? ExportDateType.POSTED_AT : null;
 
-      // Determine the excluded date based on export Type
-      const excludedPostedAtOption = !isCoreCCCModule ? ExportDateType.POSTED_AT : null;
+    // Array of unwanted dates
+    const dateOptionsToBeExcluded = [excludedSpendDateOption, excludedPostedAtOption].concat(excludedApprovedOrVerifiedOption);
 
-      // Array of unwanted dates
-      const dateOptionsToBeExcluded = [excludedSpendDateOption, excludedPostedAtOption].concat(excludedApprovedOrVerifiedOption);
+    // Get base date options
+    const unfilteredDateOptions = this.getExpenseGroupingDateOptions();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)